src/main/java/appeng/integration/modules/Mekanism.java

```diff
@@ -24,25 +24,31 @@
 
 import cpw.mods.fml.common.event.FMLInterModComms;
 
+import appeng.helpers.Reflected;
 import appeng.integration.BaseModule;
+import appeng.integration.IntegrationHelper;
 import appeng.integration.abstraction.IMekanism;
 
 
 public final class Mekanism extends BaseModule implements IMekanism
 {
-
+	@Reflected
 	public static Mekanism instance;
 
+	@Reflected
+	public Mekanism()
+	{
+		IntegrationHelper.testClassExistence( this, mekanism.api.energy.IStrictEnergyAcceptor.class );
+	}
+
 	@Override
 	public void init() throws Throwable
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)